### ISSUE TITLE
Your wordpress server appears to be overloaded.

### ISSUE THREAD

**Issue by anicioalexandre** [pristas-peter/wp-graphql-gutenberg#73]
Hi guys!

I'm using Gatsby with Wordpress and GraphQL + gutenberg to query data. But I'm getting this error when building the development server (`gatsby develop)`.

```
 ERROR #gatsby-source-wordpress-experimental_111007 

 gatsby-source-wordpress  Your wordpress server at http://localwp/graphql appears to be overloaded.

Try reducing the requestConcurrency for content updates or the previewRequestConcurrency for previews:

{
  resolve: 'gatsby-source-wordpress-experimental',
  options: {
    schema: {
      requestConcurrency: 5, // currently set to 5
      previewRequestConcurrency: 2, // currently set to 2
    }
  },
}

The GATSBY_CONCURRENT_REQUEST environment variable is no longer used to control concurrency.
If you were previously using that, you'll need to use the settings above instead.
```

I disabled WP GraphQL Gutenberg (v0.3.5) and it started to work again. This plugin might be overloading the wordpress server. It's probably some incompatibility between `WP GraphQL (v1.1.0)` and ` WP GraphQL Gutenberg (v0.3.5)`.

Appreciate any help!

**Comment by jaydiablo** [comment]
There was an issue with `0.3.5` of this plugin and >`1.0.5` of WP GraphQL that was causing an error (see #72). I believe it's been fixed in `0.3.7` of this plugin though (I just tested this in my install and it does seem to be resolved).

**Comment by anicioalexandre** [comment]
@jaydiablo thanks for replying! I've upgraded to `v0.3.7` and it was solved here too :)
